client/me/index.js

```diff
@@ -2,6 +2,8 @@ import page from '@automattic/calypso-router';
 import { makeLayout, render as clientRender } from 'calypso/controller';
 import * as controller from './controller';
 
+import './dotcom-nav-redesign-v2.scss';
```

Comment:
I initially thought that all the `dotcom-nav-redesign-v2 ` redesign CSS could live here. If we ever do another redesign, this file could be deleted and start from a "cleaner" state instead of the continuous flow of overrides.

On second thought, might be hard to communicate that, so let's do style.scss